CMakeLists.txt

```diff
@@ -226,7 +226,8 @@ set(SQLB_TSS
 	"${CMAKE_SOURCE_DIR}/src/translations/sqlb_es_ES.ts"
 	"${CMAKE_SOURCE_DIR}/src/translations/sqlb_fr.ts"
 	"${CMAKE_SOURCE_DIR}/src/translations/sqlb_ru.ts"
-	"${CMAKE_SOURCE_DIR}/src/translations/sqlb_pt_BR.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_pl.ts"
```

Comment:
Seems like a spacing issue here.  Would you be ok to match the existing style? :smile: